src/main/java/com/json_news_item/JSONContainer.java

```diff
@@ -1,27 +1,41 @@
 package com.json_news_item;
 
 
-import org.json.JSONObject;
-
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
 import java.io.PrintWriter;
-import java.util.ArrayList;
 
 public class JSONContainer {
-    private static ArrayList<JSONObject> jsonNewsList = new ArrayList<JSONObject>();
+
+    private static JSONArray jsonNewsList = new JSONArray();
 
     public static void addNewJsonRecord(JSONObject rec) {
         jsonNewsList.add(rec);
     }
 
-    public static ArrayList<JSONObject> getJsonNewsList() {
+    public JSONArray getJsonNewsList() {
         return jsonNewsList;
     }
 
     public static void displayJsonObjects(PrintWriter out) {
-        for (int i = 0; i < jsonNewsList.size(); i++) {
-            out.println(jsonNewsList.get(i));
-        }
-        out.close();
 
+
+        out.print(jsonNewsList.toJSONString());
+    }
+
+    public static String encodeHtmlTag(String tag) {
```

Comment:
I tried to use it,but this class for encoding URL, but I just need to encode HTML tags(for just simple displaying as text symbol). And this class encode all of it in symbol format "%xy", so this class doesn't do what I need.